content/docs/gto/get-started-dvc.md

```diff
@@ -0,0 +1,115 @@
+# Get Started DVC
+
+To leverage concepts of Model and Data Registries in a more explicit way, you
+can denote the `type` of each output. This will let you browse models and data
+separately, address them by `name` in `dvc get`, and eventually, see them in DVC
+Studio.
+
+Let's start with marking an artifact as data or model.
+
+If you're using `dvc add` to track your artifact, you'll need to run:
+
+```dvc
+$ dvc add mymodel.pkl --type model
+```
+
+If you're producing your models in DVC pipeline, you'll need to add
+`type: model` to `dvc.yaml` instead:
+
+```yaml
+stages:
+  train:
+    cmd: python train.py
+    deps:
+      - data.xml
+    outs:
+      - mymodel.pkl:
+          type: model # like this
+```
+
+You can also specify that while using DVCLive:
+
+```py
+live.log_artifact(artifact, "path", type="model")
+```
+
+This will make them appear in DVC Model Registry:
```

Comment:
```suggestion
This will make them appear in [Studio Model Registry](https://dvc.org/doc/studio/user-guide/model-registry/what-is-a-model-registry):
```

Reply:
Thanks! Overall, let's keep the review scope to the level of ideas and user experience. We don't even know if this will be a separate page in DVC docs, or maybe we integrate it with some other page.

Reply:
yeah this is a nitpick 😉 but was hard for me to pass the opportunity and suggest